testfm/constants.py

```diff
@@ -1,17 +1,25 @@
-import configparser
+from dynaconf import LazySettings
 
-config = configparser.ConfigParser()
-config.read("testfm.properties")
+settings = LazySettings(
+    envvar_prefix="TesTFM",
+    core_loaders=["YAML"],
+    settings_file=["settings.yaml"],
+    preload=["*.yaml"],
```

Comment:
You dont need preload, since you need only `settings.yaml`, remove:
```suggestion
```

Reply:
Want to preload `conf/*` like robottelo.